app/package.json

```diff
@@ -212,7 +212,6 @@
     "react-responsive": "^4.1.0",
     "react-router-dom": "^5.0.1",
     "react-scrollable-anchor": "^0.6.1",
-    "react-scrollspy": "^3.4.0",
```

Comment:
@fonty1 I merged #712 that's using `react-scrollspy` so please leave this in :)

Reply:
😛I shouldn't have touched it in the first place, good little library!